components/tutorials/TutorialsTile.tsx

```diff
@@ -11,11 +11,11 @@ export function TutorialsTile(props: { tutorial: TutorialClass }) {
       href={'tutorials/' + props.tutorial.slug}
     >
       <div
-        className="bg-center bg-cover flex justify-center"
+        className="bg-center bg-cover flex justify-center w-full"
         style={{
           backgroundImage: `url('tutorials/${props.tutorial.tileImage}')`,
-          width: calcRem(280),
-          height: calcRem(170)
+          height: calcRem(190),
+          margin: '0 auto'
```

Comment:
Does this `margin: '0 auto'` actually do anything? When I delete it, nothing changes.

If yes, replace with tailwindcss https://tailwindcss.com/docs/margin